File: SingleAPIExample/go/Approve.go
package main

import (
	"context"
	"fmt"

	"github.com/splitit/splitit.sdks.go"
)

func Approve(ipn string) {
	ctx := context.TODO()

	approvePaymentResp, _, err := apiClient.InstallmentPlanApi.InstallmentPlanApprove(
		ctx,
		splitit.ApproveInstallmentPlanRequest{
			InstallmentPlanNumber: ipn,
		},
	)
	if err != nil {
		fmt.Printf("InstallmentPlanApprove call failed: %s\n", err)
		return
	}
	fmt.Printf("Plan approved, Installment Plan Number: %s\n", approvePaymentResp.InstallmentPlan.InstallmentPlanNumber)
}

File: SingleAPIExample/TSNode/Approve.ts
import { InstallmentPlanApi } from 'splitit-sdk-nodejs'

let planApi: InstallmentPlanApi

export async function approve(installmentPlanNumber: string) {
  const { body: approveResponse } = await planApi.installmentPlanApprove({
    installmentPlanNumber,
    areTermsAndConditionsApproved: true,
  })
}
